subprojects/wrapper/wrapper.gradle

```diff
@@ -40,6 +41,10 @@ tasks.register("executableJar", Jar) {
     archiveName = 'gradle-wrapper.jar'
     fileMode = 0644
     dirMode = 0755
+    manifest {
+        attributes.remove(Attributes.Name.IMPLEMENTATION_VERSION.toString())
+        attributes([(Attributes.Name.IMPLEMENTATION_TITLE.toString()): "Gradle Wrapper"])
```

Comment:
We could also leave out implementation title. `Gradle Wrapper` is fine with me as well.